packages/voice/package.json

```diff
@@ -3,7 +3,7 @@
 	"version": "0.13.0",
 	"description": "Implementation of the Discord Voice API for node.js",
 	"scripts": {
-		"build": "tsup",
+		"build": "tsup && node scripts/postbuild.mjs",
```

Comment:
Use tsup banner feature instead of doing it after build

Reply:
At one point (in the old voice repository) it was done that way, but the [most recent (valid) commit in main](https://github.com/discordjs/discord.js/commit/ed94b3d9ec48bbc75859ba57b85be2ae02fa0f7e#diff-acaff049604f7eca95d252c3c1e5b40c1c05a06c6e5ec7cad5c4ac616fe4f39b) does it this way with the postbuild script. 

It looks like iCrawl explicitly changed from the tsup banner method to this postbuild script in [this commit](https://github.com/discordjs/voice/commit/644af9579f02724c489514f482640b8413d2c305) (in the old repo) for some reason. I can't tell how important it was from the immediate context though.

Regardless, I don't think it's my discretion.